src/util/tools.h

```diff
@@ -338,8 +338,7 @@ std::string sockAddrGetNameInfo(const struct sockaddr* sa);
 
 #ifdef SOPCAST
 /// \brief Finds a free port between range_min and range_max on localhost.
-int find_local_port(unsigned short range_min,
-    unsigned short range_max);
+int find_local_port(uint16_t range_min, uint16_t range_max);
```

Comment:
I'd like to avoid using exact sizes in code, so I recommend a
`typedef uint16_t network_port_t;`
e.g. in `tool.h` and using that instead. Makes our code a lot more readable.

Reply:
Done.

Reply:
I thaught about all of those, because they are ports!